content/docs/guides/ads_on_amp/ads_tips@es.md

```diff
@@ -0,0 +1,43 @@
+---
+$title : Tips para Anuncios sobre páginas AMP
+$order : 1
+---
+
+### Colocación & Controles 
```

Comment:
Intentional.

The `###` in markdown is for headlines `<h3>` in this case, and these headlines with `<br />` ? why a `<br />` tag into a headline? :-1: